recipes/pygram11/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pygram11" %}
+{% set version = "0.1.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b4e08d641d7e555a3af55d021be4d148c63439d342e800bc30ed60bb7a49e345
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+  host:
+    - llvm-openmp  # [osx]
+    - python
+    - pip
+    - setuptools >=39.0.1
+    - pybind11 >=2.2
+    - numpy >=1.12
+  run:
+    - llvm-openmp  # [osx]
+    - numpy >=1.12
```

Comment:
See the docs for how to build against numpy with a version constraint. 

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy